README.md

```diff
@@ -1,37 +1,17 @@
-# p5-template
+# p5-brush
```

Comment:
This updated README is perfect! you've definitely figured out the right vibe.

Reply:
Thank you for all of your feedback, Ashwin! I appreciate your insights and I'll try experimenting with the comments in my code. :)